waku/persistence/sqlite/nwaku.go

```diff
@@ -0,0 +1,82 @@
+package sqlite
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+)
+
+const minSupportedNWAKUversion = 8
+const maxSupportedNWAKUversion = 8
+
+func handleNWakuPreMigration(db *sql.DB) (bool, error) {
```

Comment:
If i am not wrong, this code is for scenarios where go-waku is running as service-node and not used as a client SDK(as in status-desktop). 
If someone is running an nwaku node and wants to migrate to go-waku node, then this would be useful as the database need not be cleaned.
@richard-ramos can confirm

Reply:
Correct. It would be for such cases. Although ideally they would migrate to postgresql instead.